.github/workflows/run-unit-tests.yml

```diff
@@ -26,6 +26,9 @@ jobs:
       - name: test:coverage:jest
         run: yarn test:coverage:jest
 
+      - name: test:coverage:webpack
```

Comment:
This will probably require it's on step/job/workflow now that we have parallelism.

Reply:
I think this is all fixed now.